project/build.scala

```diff
@@ -125,12 +136,12 @@ object BananaRdfBuild extends Build {
   lazy val rdfTestSuite_jvm = rdfTestSuiteM
     .project(Jvm, rdfTestSuite_common_jvm)
     .settings(Seq(resourceDirectory in Test := baseDirectory.value / "src/main/resources"):_*)
-    .dependsOn(rdf_jvm)
+    .dependsOn(rdf_jvm, ntriples_jvm)
```

Comment:
I've checked, deleting other dependencies from ntriples break the code (incl. test suite)